tests/integration/test_server.py

```diff
@@ -70,6 +70,45 @@ def test_client_emits_and_receives_message_successfully(
     assert received_args[0] == message_to_echo
 
 
+def test_client_emitting_tuple_vs_array(
```

Comment:
This integration test asserts both directions of emitting tuple/array payloads, as in: the event handlers publish array and tuple messages back to the client.